src/Files.App/Behaviors/StickyHeaderBehavior.cs

```diff
@@ -1,7 +1,3 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
```

Comment:
Are we able to remove this?

Reply:
MIT License let us use source codes under the MIT License without no notice even if copied from somewhere.

Reply:
Thanks